astropy/table/table.py

```diff
@@ -195,11 +203,28 @@ def as_array(self):
         if len(self.columns) == 0:
             return None
 
+        sys_byteorder = ('>', '<')[sys.byteorder == 'little']
```

Comment:
I like this syntax because it shows the two possibilities side by side rather at other ends of the expression (it's maybe a little confusing that the false case is listed first).